conans/test/integration/command/remote_test.py

```diff
@@ -335,6 +335,7 @@ def test_add_wrong_conancenter():
     c.run("remote add whatever https://conan.io/center", assert_error=True)
     assert "Wrong ConanCenter remote URL. You are adding the web https://conan.io/center" in c.out
     assert "the correct remote API is https://center.conan.io" in c.out
+    c.run("remote add conancenter htpps://center.conan.io")
```

Comment:
If the typo is intentional, maybe a comment specifying such?

Reply:
Not intentional, I just needed `conancenter` remote name to exist otherwise the `conan update` fails because not existing